docs/6.x/pack.md

```diff
@@ -912,3 +912,32 @@ When building a Cluster Image, `tele build` will discover `custom-planet:1.0.0`
 Docker image and vendor it along with other dependencies. During the Cluster
 installation all nodes with the role `worker` will use the custom "planet" Docker
 image instead of the default one.
+
+## Support For Intermediate Upgrades
+
+Kubernetes has a strict version skew [support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-version-skew) which makes
+cluster upgrades with versions of Kubernetes being more than a couple of minor versions apart complicated if not impossible. This can become a big issue
+if clusters are not kept constantly up-to-date.
+
+Before, the way to upgrade such a cluster had been upgrading by using cluster images with compatible Kubernetes versions in lock step
```

Comment:
Nit: IMO this paragraph reads like an excuse for this feature to exist and belongs more to the design document's "rationale" section than to the user guide. I would either try to rephrase it so it explains how upgrade works normally (i.e. you can go only from 5.0->5.2 or 5.2->5.5) or just remove it.